### PR TITLE
Troubleshooting: Add styling for `code` and `pre`

### PR DESCRIPTION
**Pull request by andyg0808**
This adds styles for the `code` and `pre` tags.

They now look about like the existing wikis do; the colors are a bit different, because I'm now using a core-primitives color:

image
https://www.cominor.com/Wiki/troubleshooting-test

image
https://react-commerce-git-troubleshooting-add-code-styling-ifixit.vercel.app/Vulcan/troubleshooting-test

## QA Notes
I'd appreciate review from @mmarcon91, because I'm changing the color a bit. The actual QA should be really straightforward (links are provided above).

Closes https://github.com/iFixit/ifixit/issues/46386

### CODE REVIEW

**Comment by ardelato** [comment]
huh, it looks like Pulldasher didn't pick up my review comment.

CR 👍 , but deploy_block 🎨 on minor suggestion.

**Comment by andyg0808** [comment]
@ardelato I'm suspicious that you might not have submitted the review, because I'm not seeing any review comment myself:

<details>
<summary>Screenshot</summary>

image

</details>

**Review by ardelato — APPROVED** [review]

File: frontend/templates/troubleshooting/prerendered.tsx [hunk 1]
```diff
@@ -44,6 +44,13 @@ const renderStyles: SystemStyleObject = {
    'a:hover': {
       textDecoration: 'underline',
    },
+
+   'code, pre': {
+      backgroundColor: 'gray.200',
+      borderRadius: '4px',
+      color: 'coolGray.600',
+      padding: '2px 4px',
```
[the remark this answers is not in the document]
Comment (ardelato) [reply frontend/templates/troubleshooting/prerendered.tsx:52]: @andyg0808 are you able to see it now?

[the remark this answers is not in the document]
Comment (andyg0808) [reply frontend/templates/troubleshooting/prerendered.tsx:52]: Oh, nice catch; thanks!

**Review by ardelato — DISMISSED** [review]
CR 👍 un_deploy_block ⚡

**Comment by jordycosta** [comment]
QA 🟢 

Styles for `code` and `pre` look good to me 👍 

styles

**Comment by mmarcon91** [comment]
> deploy_block 🟨 on a review from @mmarcon91

For me it's ok 👍

**Comment by andyg0808** [comment]
QA :+1: carryover.

**Review by ardelato — COMMENTED** [review]
CR 👍

**Review by ardelato — APPROVED** [review]

**Pull request merged**